website/moneybirdsynchronization/management/commands/contactsync.py

```diff
@@ -0,0 +1,45 @@
+from django.core.management.base import BaseCommand
+
+from moneybirdsynchronization.administration import HttpsAdministration
+from moneybirdsynchronization.models import Contact
+from thaliawebsite import settings
+from members.models import Member
+from django.db.models import Q, Subquery, OuterRef
+
+class Command(BaseCommand):
+    """This command can be executed periodically to sync the contacts with MoneyBird."""
+
+    def handle(self, *args, **options):
+        members_without_contact = Member.objects.filter(~Q(id__in=Subquery(Contact.objects.filter(member=OuterRef('pk')).values('member'))))
+
```

Comment:
It would be better to put the actual syncing in `services.py` and call it from this command. This way, it is easier to implement tests and it becomes possible to call the sync in a different way than just via the management command